website/pages/en/help.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+const React = require('react');
+
+const CompLibrary = require('../../core/CompLibrary.js');
+const Container = CompLibrary.Container;
+const GridBlock = CompLibrary.GridBlock;
+
+const translate = require('../../server/translate.js').translate;
```

Comment:
Is this file generated somehow? It's deleted in this PR.

Reply:
This is a file from Docusaurus.

Reply:
Ok. I'd expect something like:
```js
const {translate} = require('docusaurus/server');
```
but not sure how this thing works currently.